bundles/org.openhab.ui.homebuilder/USAGE.md

```diff
@@ -1,18 +1,116 @@
-# Home Builder
+---
+layout: documentation
+title: Home Builder
+source: https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.ui.homebuilder/USAGE.md
+---
 
-> Boilerplate for the [Items](http://docs.openhab.org/configuration/items.html), [sitemap](http://docs.openhab.org/configuration/sitemaps.html) files and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard.
+{% include base.html %}
 
-## Items
+<!-- Attention authors: Do not edit directly. Please add your changes to the appropriate source repository -->
 
-The tool allows you to generate openHAB Items for your home structure.
-You can choose to generate textual `*.items` file content or construct a request directly to the REST API that'll create the items for you.
+# Home Builder
 
+Boilerplate for the [Items](http://docs.openhab.org/configuration/items.html), [sitemap](http://docs.openhab.org/configuration/sitemaps.html) files and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard.
```

Comment:
@kubawolanin You should actually use https instead of http for all links to docs.openhab.org!